tests/tizen_capi/unittest_tizen_capi.cc

```diff
@@ -1728,6 +1728,69 @@ TEST (nnstreamer_capi_util, compare_info)
   EXPECT_EQ (status, ML_ERROR_NONE);
 }
 
+/**
+ * @brief Test utility functions (public)
+ */
+TEST (nnstreamer_capi_util, info_create_1_n)
+{
+  int status = ml_tensors_info_create (nullptr);
+  ASSERT_EQ (status, ML_ERROR_INVALID_PARAMETER);
+}
+
+extern int
+ml_tensors_info_create_from_gst (ml_tensors_info_h * ml_info,
```

Comment:
IMO, this is unnecessary.
line 15 #include <nnstreamer-capi-private.h> for internal test

Reply:
Thanks!